bench-tps/src/bench.rs

```diff
@@ -611,14 +619,23 @@ pub fn generate_and_fund_keypairs<T: Client>(
 
     if lamports_per_account > last_keypair_balance {
         let (_, fee_calculator) = client.get_recent_blockhash().unwrap();
-        let extra =
+        let account_desired_balance =
             lamports_per_account - last_keypair_balance + fee_calculator.max_lamports_per_signature;
-        let total = extra * (1 + keypairs.len() as u64);
+        let extra: u64 = (0..depth).map(|i| 4u64.pow(i)).sum();
```

Comment:
Isn't it computing the same as total_keys from generate_keypairs?

Reply:
Yes, minus the last spread. I'll pass from generate_keypairs instead.